src/dmd/dsymbolsem.d

```diff
@@ -1145,10 +1145,12 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
             sc.protection = imp.protection;
             for (size_t i = 0; i < imp.aliasdecls.dim; i++)
             {
+                bool flag;
                 AliasDeclaration ad = imp.aliasdecls[i];
                 //printf("\tImport %s alias %s = %s, scope = %p\n", toPrettyChars(), aliases[i].toChars(), names[i].toChars(), ad._scope);
-                if (imp.mod.search(imp.loc, imp.names[i]))
+                if (imp.mod.search(imp.loc, imp.names[i]) /*, IgnorePrivateImports*/)
```

Comment:
Why is this comment here?

Reply:
When the deprecation is over, that argument should ne uncommented

Reply:
Ok.